packages/grafana-ui/src/components/Table/Table.tsx

```diff
@@ -91,7 +91,7 @@ function useTableStateReducer(props: Props) {
 
       return newState;
     },
-    [props.onColumnResize, props.onSortByChange, props.data]
+    [props]
```

Comment:
nit: Best practices is to destruct nested props and use them in the dependencies.

Reply:
done